src/components/Modals/Modal/index.tsx

```diff
@@ -13,7 +13,7 @@ const Modal: FunctionComponent<{
 			<div className="modal__overlay" onClick={onClose} />
 			<div className="modal__body">
 				<div className="modal__close" onClick={onClose}>
-					<CloseIcon />
+					<img src={CloseIcon} alt="Close Modal" />
```

Comment:
I would prefer to keep it as SVG. It will give us better control over styling, animations and so on.
Why did you transform that to plain img, what's the reason behind it?